lib/smart_answer_flows/child-benefit-tax-calculator.rb

```diff
@@ -0,0 +1,18 @@
+module SmartAnswer
+  class ChildBenefitTaxCalculatorFlow < Flow
+    def define
+      name "child-benefit-tax-calculator"
+      start_page_content_id "201fff60-1cad-4d91-a5bf-d7754b866b87"
```

Comment:
```suggestion
      start_page_content_id "0e1de8f1-9909-4e45-a6a3-bffe95470275"
```

I think it would be good if [this matches the current `content_id`](https://github.com/alphagov/publisher/pull/629/files#diff-d75e054d219dcab61db1959fca8b4f9fR25) so we keep the history in the Publishing API. It's currently published by Publisher, but it'll end up being published by Smart Answers once this is merged in.

Reply:
Thanks @thomasleese ! We have made this change in https://github.com/alphagov/smart-answers/pull/4762/commits/d324d7156b06b8b1114f68e6bd3c62241053169e